app.go

```diff
@@ -126,6 +126,7 @@ func serve(port int, hc *resources.MapperHealthcheck, cth *resources.ContentTran
 		hc.ProducerQueueCheck(),
 	)
 	r.Handle("/content-transform/{uuid}", cth).Methods("POST")
```

Comment:
I renamed ContentTransformHandler into MapEndpointHandler. It is necessary to keep both endpoints for now because I will not be able to release the PAM changes and these changes simultaneously (they live in different clusters).